src/components/data_submitter/DataAccessGovernance.js

```diff
@@ -0,0 +1,168 @@
+import ConsentGroup from './ConsentGroup';
+import { useEffect, useState } from 'react';
+import { cloneDeep, isNil } from 'lodash/fp';
+import { div, h, a, span} from 'react-hyperscript-helpers';
+import SimpleButton from '../SimpleButton';
+import { Theme } from '../../libs/theme';
+import { DAC } from '../../libs/ajax';
+import {SearchSelect} from '../SearchSelect';
+import {RadioButton} from '../RadioButton';
+
+const OPEN_ACCESS = "Open Access";
+const CLOSED_ACCESS = "Closed Access";
```

Comment:
Codacy has a fix for the issue: [Strings must use singlequote.](https://app.codacy.com/gh/DataBiosphere/duos-ui/pullRequest?prid=9969451)
```suggestion
const CLOSED_ACCESS = 'Closed Access';
```